recommendation/project.all.json

```diff
@@ -82,6 +82,12 @@
               "Type": "ProjectFile",
               "Value": "",
               "Description": "Migrate csproj file to core"
+            },
+            {
+              "Name": "WebFormsPorting",
+              "Type": "ProjectType",
```

Comment:
Is ProjectType a new action type we're adding?

Reply:
Yes, I added a new action type called project type. I figured WCF and any other stuff that would otherwise be used with a project rewriter could be put under this action type